meson.build

```diff
@@ -2659,6 +2659,15 @@ if conf.get('ENABLE_OOMD') == 1
                    install_rpath : rootlibexecdir,
                    install : true,
                    install_dir : rootlibexecdir)
+
+        executable('oomctl',
```

Comment:
hmm, i wonder about the name. so far we used non-prefixed only for super-primary entrypoint calls, more exotic stuff got a `systemd-` prefix, to not pollute the namespace too much. I have the suspicion this should be `systemd-oom`?

Reply:
is this my chance to suggest `systoomctl`? I really like this portmanteau

Reply:
I think that as long as this is in rootlibexecdir, the name is secondary. Let's leave the name as is.

But please make this
```
        public_programs += executable(...
```
so that we test the `--help` output.